public/bin/linux/bottles-wine

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
Why we need a shell script? Can we not run it via typescript?
What is the problem not running it from heroic itself?